map/framework.hpp

```diff
@@ -591,6 +591,10 @@ class Framework
   bool IsOnRoute() const { return m_routingSession.IsOnRoute(); }
   bool IsRouteNavigable() const { return m_routingSession.IsNavigable(); }
   void BuildRoute(m2::PointD const & start, m2::PointD const & finish, uint32_t timeoutSec);
+  /// There were a bug inside FollowRoute method. Router follows route even it wasn't call.
```

Comment:
There was a bug inside FollowRoute method. Router follows route even if it is not called.